src/loki/values/values.yaml

```diff
@@ -50,6 +60,12 @@ test:
   enabled: false
 # Configuration for the write pod(s)
 write:
+  # Remove default anti-affinity since this is a 1-replica setup
+  affinity: ""
+
+  # replicas to 1 to save resources
+  replicas: 1
+
```

Comment:
I think I'd rather have this HA by default and drop the resources blocks from our values. Resources are more likely to need tweaking per environment, but >1 replica is likely to be a need everywhere so I think its good to ensure we are testing multi-replica setups. Dropping default resource values does makes HPA a bigger effort to setup, but I think its the right move given how environment specific that tends to be.

Reply:
For gitlab i left the "real" setup as default and used variable overrides + a prebuilt uds-config.yaml for the "dev/ci/lite" set of values. I think that's a good pattern to follow for all the things. Thoughts?

Reply:
Yeah agreed here on that - think the tricky part here is single capability which doesn't exist in gitlab + figuring out how/where we want to validate the prod setup.

Reply:
Technically gitlab is a single package test, but yeah the deploy test running in a bundle vs in a zarf package prevents getting around the issue with overrides. 🫠

Reply:
I'm also a big fan of nightly "full" prod config test. RKE2 for now, potentially down the road rke2, eks, etc.

That will allow us to keep the frequent CI tests lightweight and quick still getting the feedback and confidence in the full deployment without large compute cost implications.